packages/scandipwa/src/component/FieldFile/FieldFile.container.js

```diff
@@ -55,8 +55,17 @@ export class FieldFileContainer extends PureComponent {
         if (this.fieldRef) {
             const { files } = this.fieldRef;
             this.setState({ isLoading: true });
+            const { name } = files[0] ? files[0] : '';
```

Comment:
In case if files[0] is undefined you will be getting undefiend in name, because 
` const { name } = '';

So just `files[0] || {}`